ts/components/dialog/EditProfileDialog.tsx

```diff
@@ -51,8 +52,8 @@ export class EditProfileDialog extends React.Component<{}, State> {
     this.convo = getConversationController().get(UserUtils.getOurPubKeyStrFromCache());
 
     this.state = {
-      profileName: this.convo.getProfileName() || '',
-      setProfileName: this.convo.getProfileName() || '',
+      profileName: this.convo.getRealSessionUsername() || '',
```

Comment:
I originally was going to suggest using`profileName` instead of `displayNameInProfile` but I see that is already used. Maybe it's best to not change it. I just found it confusing at first.

Reply:
yep I agree the name is not perfect. I kind of like having a string easily searchable for that field, but we can know very easily make a migration to change it if needed!
`profileName` is not right neither as this field also holds the name of the `opengroup/closedgroup`

Reply:
Cool in that case lets keep it as is.